api/src/integrations/epic/orders-utils.js

```diff
@@ -62,6 +62,41 @@ class Orders extends BaseRecordUtils {
       }
     };
   }
+
+  /**
+   * Indicates if a record is a fee order or not.
+   * 
+   * @param {object} transformedRecord Epic record that has been transformed to NRPTI format
+   * @returns {boolean} Indication if the record is a fee order
+   * @memberof Orders
+   */
+  isRecordFeeOrder(transformedRecord) {
```

Comment:
Does this also need to be added to the inspections utils, if the fee orders sometimes come in from epic as inspections?

Reply:
Great catch! Yes, I forgot to add it to the Inspection. I think I might add it to the base record just so all Epic types have access should they need it. Really there should be no Fee Orders arriving no matter what type they have been categorized as.